examples/basic/variables.tf

```diff
@@ -76,6 +76,48 @@ variable "ibmcloud_api_key" {
   type        = string
 }
 
+variable "access_host_or_ip" {
+  description = "Jump/Access server public host name or IP address. This host name/IP is used to reach the landscape."
+  type        = string
+  default     = "not_used"
+}
+
+variable "private_services_host_or_ip" {
+  description = "Private IP address where management services should be configured. Not used here."
+  type        = string
+  default     = "not_used"
+}
+
+variable "internet_services_host_or_ip" {
+  description = "Private IP address where internet services (like proxy) should be configured. Not used here."
+  type        = string
+  default     = "not_used"
+}
+
+variable "configure_proxy" {
```

Comment:
Remove all inputs that are not designed to be changed. Make them locals instead if needed.

Reply:
need to think about this, as you want to execute the whole module, including os initialization

Reply:
Set it a locale in this scenario - the variable description currently indicate to not touch its value "Do not configure proxy in this example by default."

Reply:
Still an issue

Reply:
DOne